src/app/config.ts

```diff
@@ -5,9 +5,14 @@ export const config = {
   metadata: {
     title: 'LUKSO dApp Boilerplate',
     description: 'Open Source LUKSO Dev Stack',
+    url: 'https://boilerplate.lukso.tech/',
+    icon: '/lyx_token_symbol.svg',
   },
   extension: {
     name: 'Universal Profiles',
     url: 'https://chrome.google.com/webstore/detail/universal-profiles/abpickdkkbnbcoepogfhkhennhfhehfn?hl=en',
   },
+  walletTools: {
+    walletConnectProjectID: 'f38002254154fd4b603998e31923bfcc',
```

Comment:
You have to provide a project ID when using Web3Modal. This must be created on Wallet Connect's end and should change for every dApp that implements Web3Modal. Must be provided, otherwise, the window won't get instantiated.

Reply:
https://docs.walletconnect.com/web3modal/javascript/about

Reply:
It would be better to put this ProjectID as a environment variable injected from next.config.js. Something like .env NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID for example.